web/annotation_layer_builder.css

```diff
@@ -28,6 +29,7 @@
     --input-unfocused-border-color: ActiveText;
     --input-disabled-border-color: GrayText;
     --input-hover-border-color: Highlight;
+    --link-border: 2px solid LinkText;
```

Comment:
For the `outline` rules just below we're using `1.5px`, do we perhaps want to use that value for this new border variable as well?
(I obviously don't know the a11y side of things, but just a question about the general consistency here.)